gui/window_character_lifestyle.gui

```diff
@@ -1368,32 +1376,245 @@ window = {
+										}
+										blockoverride "spiritprogressbartexture"
+										{
+											texture = "gfx/interface/progressbars/war_progress_blue.dds"
+										}
+										blockoverride "decayprogressbartexture"
+										{
+											texture = "gfx/interface/progressbars/progress_water_decay.dds"
+										}
+									}
+
+									hbox = {
+										text_multi = {
+											align = center
+											layoutpolicy_horizontal = expanding
+											autoresize = yes
+											text = "WC_LIFESTYLE_WINDOW_ELEMENTAL_WATER_SPIRITDECAY_PROGRESSBAR_RIGHT_TEXT"
+										}
+									}
+								}
+								hbox = {
+									text_multi = {
+										align = center
+										autoresize = yes
+										text = "WC_LIFESTYLE_WINDOW_ELEMENTAL_WATER_SPIRITDECAY_PROGRESSBAR_BOTTOM_TEXT"
+									}
+								}
+							}
+							vbox = {
+								visible = "[EqualTo_string( Lifestyle.GetKey, 'elemental_air_magic_lifestyle' )]"
+								layoutpolicy_horizontal = expanding
+
+								hbox = { # Air progressbar
+									layoutpolicy_horizontal = expanding
+
+									margin_top = 10
+
+									hbox = {
+										text_multi = {
+											align = center
+											layoutpolicy_horizontal = expanding
+											autoresize = yes
+											text = "WC_LIFESTYLE_WINDOW_ELEMENTAL_AIR_SPIRITDECAY_PROGRESSBAR_LEFT_TEXT"
+										}
+									}
+
+									elementalprogressbar = {
+										blockoverride "layoutstretchfactor_horizontal_progressbarvaluemaxscaled"
+										{
+											layoutstretchfactor_horizontal = "[GetProgressBarValueMaxScaled( FixedPointToFloat(GuiScope.SetRoot( GetPlayer.MakeScope ).ScriptValue('wc_layoutstretchfactor_elemental_air_spiritanddecay_favour_value')), '(float)100', '(int32)350' )]"
+										}
+										blockoverride "layoutstretchfactor_horizontal_progressbarvaluemaxotherscaled"
+										{
+											layoutstretchfactor_horizontal = "[GetProgressBarValueMaxOtherScaled( FixedPointToFloat(GuiScope.SetRoot( GetPlayer.MakeScope ).ScriptValue('wc_layoutstretchfactor_elemental_air_spiritanddecay_favour_value')), '(float)100', '(int32)350' )]"
+										}
+										blockoverride "spiritprogressbartexture"
+										{
+											texture = "gfx/interface/progressbars/progress_air_spirit.dds"
+										}
+										blockoverride "decayprogressbartexture"
+										{
+											texture = "gfx/interface/progressbars/progress_air_decay.dds"
+										}
+									}
+
+									hbox = {
+										text_multi = {
+											align = center
+											layoutpolicy_horizontal = expanding
+											autoresize = yes
+											text = "WC_LIFESTYLE_WINDOW_ELEMENTAL_AIR_SPIRITDECAY_PROGRESSBAR_RIGHT_TEXT"
+										}
+									}
+								}
+								hbox = {
+									text_multi = {
+										align = center
+										autoresize = yes
+										text = "WC_LIFESTYLE_WINDOW_ELEMENTAL_AIR_SPIRITDECAY_PROGRESSBAR_BOTTOM_TEXT"
```

Comment:
You don't need to have one line here.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155837917-6d0a4c3c-27ee-4133-9594-70f090b0f10a.png)

</details>

You could split it into 2 texts:
```
[wc_elemental_air_spiritdecay_magic_favour|E]: [GuiScope.SetRoot(GetPlayer.MakeScope).ScriptValue('wc_current_elemental_air_favour_value')|=+0]
```
Which doesn't depend on your elemental disposition.
And:
```
[wc_elemental_air_spirit_decay_magic_disposition|E]: Neutral
```
Which depends.